test/test_dht_storage.cpp

```diff
@@ -131,4 +132,52 @@ TORRENT_TEST(dht_storage)
 	delete s;
 }
 
+TORRENT_TEST(dht_storage_counters)
+{
+	dht_settings sett = test_settings();
+	dht_storage_interface* s = dht_default_storage_constructor(node_id(0), sett);
+
+	TEST_CHECK(s != NULL);
+
+	sha1_hash n1 = to_hash("5fbfbff10c5d6a4ec8a88e4c6ab4c28b95eee401");
+	sha1_hash n2 = to_hash("5fbfbff10c5d6a4ec8a88e4c6ab4c28b95eee402");
+	sha1_hash n3 = to_hash("5fbfbff10c5d6a4ec8a88e4c6ab4c28b95eee403");
+	sha1_hash n4 = to_hash("5fbfbff10c5d6a4ec8a88e4c6ab4c28b95eee404");
+
+	TEST_EQUAL(s->counters().peers, 0);
+	TEST_EQUAL(s->counters().torrents, 0);
+
+	tcp::endpoint p1 = tcp::endpoint(address::from_string("124.31.75.21"), 1);
```

Comment:
there's a small utility function here: https://github.com/arvidn/libtorrent/blob/master/test/test_peer_list.cpp#L51
that you might want to pull out into one of the common test files (setup_transfer.hpp for instance). not a big deal though